File: ch13/Chapter13-WCFService-CS/App_Code/IMyCustomDataContractService.cs
using System.Runtime.Serialization;
using System.ServiceModel;
          
[DataContract]
public class Customer
{
    [DataMember]
    public string Firstname;
          
    [DataMember]
    public string Lastname;
}
          
[ServiceContract(Namespace="http://jasongaylord.com/")]
public interface IMyCustomDataContractService
{
    [OperationContract]
    string HelloFirstName(Customer cust);
          
    [OperationContract]
    string HelloFullName(Customer cust);
}

File: ch13/Chapter13-WCFService-VB/App_Code/IMyCustomDataContractService.vb
Imports System.ServiceModel
Imports System.Runtime.Serialization

<DataContract()>
Public Class Customer
    <DataMember()>
    Public FirstName As String

    <DataMember()>
    Public LastName As String
End Class

<ServiceContract()>
Public Interface IMyCustomDataContractService
    <OperationContract()>
    Function HelloFirstName(ByVal cust As Customer) As String

    <OperationContract()>
    Function HelloFullName(ByVal cust As Customer) As String
End Interface